examples/code_general_impots/Section_I_Dispositions_generales.catala

```diff
@@ -0,0 +1,40 @@
+@@Code général des impôts@@
+@@Livre premier : Assiette et liquidation de l'impôt@@+
+@@Première Partie : Impôts d'État@@++
+@@Titre premier : Impôts directs et taxes assimilées@@+++
+@@Chapitre premier : Impôt sur le revenu@@++++
+@@Section I : Dispositions générales@@+++++
+@@0I : Définition du revenu net global@@++++++
+@Article 1 A@
+
+Il est établi un impôt annuel unique sur le revenu des personnes physiques désigné sous le nom d'impôt sur le revenu. Cet impôt frappe le revenu net global du contribuable déterminé conformément aux dispositions des articles 156 à 168.
+
+Ce revenu net global est constitué par le total des revenus nets des catégories suivantes :
+
+– Revenus fonciers ;
+
+– Bénéfices industriels et commerciaux ;
+
+– Rémunérations, d'une part, des gérants majoritaires des sociétés à responsabilité limitée n'ayant pas opté pour le régime fiscal des sociétés de personnes dans les conditions prévues au IV de l'article 3 du décret n° 55-594 du 20 mai 1955 modifié et des gérants des sociétés en commandite par actions et, d'autre part, des associés en nom des sociétés de personnes et des membres des sociétés en participation lorsque ces sociétés ont opté pour le régime fiscal des sociétés de capitaux ;
+
+– Bénéfices de l'exploitation agricole ;
+
+– Traitements, salaires, indemnités, émoluments, pensions et rentes viagères ;
+
+– Bénéfices des professions non commerciales et revenus y assimilés ;
+
+– Revenus de capitaux mobiliers ;
+
+– Plus-values de cession à titre onéreux de biens ou de droits de toute nature, déterminés conformément aux dispositions des articles 14 à 155, total dont sont retranchées les charges énumérées à l'article 156.
+
+/*
+déclaration structure CategoriesImpotRevenu:
+donnée revenus_fonciers contenu CategorieImpotRevenu
```

Comment:
Thanks for implementing the changes! There's only a small thing that remains: this line (as well as ther others below) shoud read 

```
donnée revenus_fonciers contenu montant
```

Indeed, what follows `contenu` is the type of the `donnée`, which here should be `montant`.

Reply:
You're welcome, thanks for being understanding about my mistakes and being pedagogical about the changes that should be implemented.

Do you want me to write a rule to total the categories in this PR? Or in a later PR?